README.md

```diff
@@ -161,6 +161,15 @@ the `encryption_*` and `ssl_*` options
 directly to the constructor as keyword arguments, even though it is
 required when they are placed in the environment file.
 
+pam (pam_password) logins
```

Comment:
Consider capitalizing "pam" and dropping "(pam_password)". The title implies this section is all about PAM related logins.

Reply:
> Consider capitalizing "pam" and dropping "(pam_password)". The title implies this section is all about PAM related logins.

sounds good

Reply:
Done.